File: hw2/cpp-hello/src/student.cpp
#include "Student.h"

using namespace std;

namespace student{
    Student::Student(string first,string last,float grade){
        m_first=first;
        m_last=last;
        m_grade=grade;
    }
    string Student::fullname(){
        return m_first+" "+ m_last;
    }

    string Student::email(){
        return m_first+"."+ m_last+"@email.com";
    }
    float Student::curvegrade(){
        return m_grade+1.0;
    }
    string Student::getfirst(){
        return m_first;
    }
    string Student::getlast(){
        return m_last;
    }
    float Student::getgrade(){
        return m_grade;
    }
}

File: hw3/student.js
class Student{
    constructor(first,last,grade){
        this._first=first
        this._last=last
        this._grade=grade
    }
    fullName(){
        return `${this._first} ${this._last}`
    }
    email(){
        return `${this._first}.${this._last}@email.com`
    }
    isPass(){
        if(this._grade>=2)
            return true
        else
            return false
    }
    get first(){
        return this._first
    }
    get last(){
        return this._last
    }
    get grade(){
        return this._grade
    }
    
}   
module.exports={'Student':Student}